src/wallet/wallet2.cpp

```diff
@@ -658,13 +660,19 @@ static uint64_t decodeRct(const rct::rctSig & rv, const crypto::public_key &pub,
   }
 }
 //----------------------------------------------------------------------------------------------------
-bool wallet2::wallet_generate_key_image_helper(const cryptonote::account_keys& ack, const crypto::public_key& tx_public_key, size_t real_output_index, cryptonote::keypair& in_ephemeral, crypto::key_image& ki)
+bool wallet2::wallet_generate_key_image_helper(const cryptonote::account_keys& ack, const crypto::public_key& tx_public_key, size_t real_output_index, cryptonote::keypair& in_ephemeral, crypto::key_image& ki) const
 {
   if (!cryptonote::generate_key_image_helper(ack, tx_public_key, real_output_index, in_ephemeral, ki))
     return false;
   return true;
 }
 //----------------------------------------------------------------------------------------------------
+bool wallet2::wallet_generate_key_image_helper_export(const cryptonote::account_keys& ack, const crypto::public_key& tx_public_key, size_t real_output_index, cryptonote::keypair& in_ephemeral, crypto::key_image& ki, size_t multisig_key_index) const
```

Comment:
Maybe I'm asking a similar question again, but why should this function and also `wallet_generate_key_image_helper` be introduced, while they work as a simple alias?

Incidentally, my pending PR #2529 (which you approved) removes `wallet2::wallet_generate_key_image_helper`.

Reply:
I guess it could be inlined manually, sure. Looking at #2529 now, I guess I approved it because I don't have strong objections against the change, and it came with a bugfix, and I tend to accept "arbitrary" changes in code which also fixes something. I'll inline it manually then, since I don't really care much either way, and it's just two lines.

Reply:
Done. It's a new commit, as it's conflicting a lot with history otherwise.